ci-operator/config/openstack-k8s-operators/openstack-operator/openstack-k8s-operators-openstack-operator-master.yaml

```diff
@@ -42,6 +46,10 @@ tests:
   commands: make gofmt || true
   container:
     from: src
+- as: precommit-check
+  commands: GOFLAGS= PRE_COMMIT_HOME=/tmp pre-commit run --all-files --show-diff-on-failure
```

Comment:
We need to add `SKIP=no-commit-to-branch` here as well as the CI will check out master branch which is protected by default locally.

Reply:
got it